internal/gen/types/types.go

```diff
@@ -19,9 +19,15 @@ import (
 const (
 	LangTypescript = "typescript"
 	LangGo         = "go"
+	LangSwift      = "swift"
 )
 
-func Run(ctx context.Context, projectId string, dbConfig pgconn.Config, lang string, schemas []string, postgrestV9Compat bool, fsys afero.Fs, options ...func(*pgx.ConnConfig)) error {
+const (
+	SwiftPublicAccessControl   = "public"
+	SwiftInternalAccessControl = "internal"
```

Comment:
I'm not an expert in swift so I want to ask if you plan to add other access control, like `private`, `open`, etc. in the future? Are they useful at all?